docs/archive/meeting-notes/ratify-weekly-notes-2022-Jul-2022-Dec.md

```diff
@@ -518,16 +518,16 @@ Recording: https://youtu.be/nSjn5_kRvoA
                 type: object
                 x-kubernetes-preserve-unknown-fields: true
 ```
-- review notary verifier sample   
- [David] Cert should be configured in a new CertStore CRD, notary verifier should contain a reference to the certStore object being configured for this verifier object.
+- review notation verifier sample   
+ [David] Cert should be configured in a new CertStore CRD, notation verifier should contain a reference to the certStore object being configured for this verifier object.
 ```
 apiVersion: batch.ratify.deislabs.io/v1alpha1
 kind: Verifier
 metadata:
   name: verifier-sample
 spec:
-  name: notaryv2
-  artifactTypes: application/vnd.cncf.notary.v2.signature
+  name: notation
+  artifactTypes: application/vnd.cncf.notary.signature
```

Comment:
it may not be relevant to updating this doc in particular, but can we double check that the artifact type for notation has moved away from using the 'v2'

Reply:
confirmed <img width="641" alt="image" src="https://github.com/deislabs/ratify/assets/74894646/02595a70-641e-4ab1-800a-5defa993add1">

@akashsinghal